src/components/forms/NetworkDevicesForm/Acls.tsx

```diff
@@ -0,0 +1,111 @@
+import type { FC } from "react";
+import type { LxdNicDevice } from "types/device";
+import type { LxdNetwork } from "types/network";
+import type { CustomNetworkDevice } from "util/formDevices";
+import type { InstanceAndProfileFormikProps } from "components/forms/instanceAndProfileFormValues";
+import ExpandableList from "components/ExpandableList";
+import ResourceLink from "components/ResourceLink";
+import NetworkAclSelector from "pages/networks/forms/NetworkAclSelector";
+
+interface Props {
+  project: string;
+  managedNetworks: LxdNetwork[];
+  device: LxdNicDevice | CustomNetworkDevice;
+  readOnly?: boolean;
+  formik?: InstanceAndProfileFormikProps;
+  index?: number;
+  canSelectManualAcls?: boolean;
+}
+
+const Acls: FC<Props> = ({
```

Comment:
I think the spacing in edit mode can be improved, maybe get input from @piperdeck 

<img width="1001" height="487" alt="image" src="https://github.com/user-attachments/assets/725fab8d-57b2-4328-8126-4f28798674fe" />